src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -36,7 +47,7 @@
     </li>
     <li>
       <p>
-        Drag <sky-code>skyux-ca.crt</sky-code> into the list of certificates.
+        Open the <sky-code>.skyux/certs</sky-code> folder in your home directory and drag <sky-code></sky-code> into the list of certificates.
```

Comment:
We are missing something here inside the `sky-code` element.

Reply:
ahh yep!